pkg/controller/restore_controller.go

```diff
@@ -284,6 +296,10 @@ func (controller *restoreController) getValidationErrors(itm *api.Restore) []str
 		validationErrors = append(validationErrors, "BackupName must be non-empty and correspond to the name of a backup in object storage.")
 	}
 
+	if errs := collections.ValidateIncludesExcludes(itm.Spec.IncludedResources, nonRestorableResources); len(errs) > 0 {
```

Comment:
This could be misleading if the error returned from ValidateIncludesExcludes is due to something other than nodes being explicitly included. Maybe rather than calling ValidateIncludesExcludes for this, just check if anything in the nonRestorable list is in IncludedResources?

Reply:
Good point, thank you.